nextflow.config

```diff
@@ -9,58 +9,119 @@
 // Global default params, used in configs
 params {
 
-    // TODO nf-core: Specify your pipeline's command line flags
     // Input options
-    input                      = null
+    input = null
 
+    // Workflow mode
+    mode = null
 
-    // References
-    genome                     = null
-    igenomes_base              = 's3://ngi-igenomes/igenomes'
-    igenomes_ignore            = false
-    // MultiQC options
-    multiqc_config             = null
-    multiqc_title              = null
-    multiqc_logo               = null
-    max_multiqc_email_size     = '25.MB'
-    multiqc_methods_description = null
+    // Process configuration
+    processes_manual  = false
+    processes_include = null
+    processes_exclude = null
 
-    // Boilerplate options
-    outdir                     = null
-    tracedir                   = "${params.outdir}/pipeline_info"
-    publish_dir_mode           = 'copy'
-    email                      = null
-    email_on_fail              = null
-    plaintext_email            = false
-    monochrome_logs            = false
-    hook_url                   = null
-    help                       = false
-    validate_params            = true
-    show_hidden_params         = false
-    schema_ignore_params       = 'genomes'
-    enable_conda               = false
+    // Reference genome information; iGenomes is effectively disabled but retained for linting
+    genome          = null
+    force_genome    = false
+    igenomes_base   = 's3://ngi-igenomes/igenomes'
+    igenomes_ignore = true
+    hmf_genome_base = 'https://pub-cf6ba01919994c3cbd354659947f74d8.r2.dev/genomes'
```

Comment:
see above about consolidating where data lives

Reply:
Totally agree with you re iGenomes and co. I've put forward some questions around this in your [above comment thread](https://github.com/nf-core/oncoanalyser/pull/29#discussion_r1680831438) to proceed, and given the amount of work / discussion to admit new data into iGenomes I think it's best to progress this piece of work independently from the first release.

Also related: https://github.com/nf-core/oncoanalyser/pull/29#discussion_r1697968888